src/AccessibilityInsights.Core/Properties/AssemblyInfo.cs

```diff
@@ -7,11 +7,11 @@
 // General Information about an assembly is controlled through the following
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("AccessibilityInsights.Core")]
+[assembly: AssemblyTitle("Axe.Windows.Core")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("AccessibilityInsights.Core")]
+[assembly: AssemblyProduct("Axe.Windows.Core")]
 [assembly: AssemblyCopyright("Copyright �  2017")]
```

Comment:
Looks like the copyright character is incorrect (was incorrect before, is still incorrect)